src/fosslight_binary/_jar_analysis.py

```diff
@@ -196,4 +197,11 @@ def analyze_jar_file(path_to_find_bin):
         logger.debug(f"Error to get depency Info in jar_contets: {ex}")
         success = False
 
+    try:
+        if json_file != "":
```

Comment:
Let's change this one.
` if json_file != "": -> if os.path.isfile(json_file ):`

Reply:
got it